lib/tasks/setup.rake

```diff
@@ -8,12 +8,12 @@ namespace :setup do
     end
   end
 
-  desc 'Create a superadmin with factory'
-  task superadmin_factory: :environment do
+  desc 'Create Superadmin account ready to use (no activation needed)'
+  task superadmin_initialized: :environment do
     if ENV['email'].present?
       FactoryGirl.create(:user, email: ENV['email'], password: 'asdfasdf', role: 'superadmin')
```

Comment:
I think it would be better to just use `User.create` here, FactoryGirl isn't loaded on production and it seems the factory doesn't actually do anything more here (it only sets these three fields as well).